src/components/shared/Entity/Actions/Save.tsx

```diff
@@ -16,15 +18,27 @@ import useNotificationStore, {
 interface Props {
     disabled: boolean;
     onFailure: Function;
-    draftEditorStoreName: DraftEditorStoreNames;
+    logEvent: CustomEvents;
     dryRun?: boolean;
+    draftEditorStoreName: DraftEditorStoreNames;
 }
 
+const trackEvent = (logEvent: Props['logEvent'], payload: any) => {
```

Comment:
Small helper function to call to LogRocket. I think eventually we should make some kind of service/map/typing/thingy where we can store the allowed events (`logEvent` down below) and their corresponding payloads.